api/handlers.go

```diff
@@ -130,7 +130,7 @@ func getImages(c *context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Look for an engine that has all the images we need.
-	for engine, images := range engineImages {
+	for _, images := range engineImages {
```

Comment:
If we're not going to use the first argument, can we avoid constructing it up above on L#127? Or maybe put `image.Engine` in there as the key.